### DISCUSSION TITLE
Welcome to XMOJ-Script Discussions!

### DISCUSSION THREAD

**Discussion by langningchen** [langningchen/XMOJ-Script#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PythonSmall-Q** [comment]
Hallow, guys. I am sean. i am mainly using this script and trying to maintain it (although my PRs are all rubbish
